standalone/src/main/java/io/pravega/local/LocalPravegaEmulator.java

```diff
@@ -22,7 +21,9 @@
     private final InProcPravegaCluster inProcPravegaCluster;
 
     @Builder
-    private LocalPravegaEmulator(int zkPort, int controllerPort, int segmentStorePort, int restServerPort, boolean enableAuth, boolean enableTls) {
+    private LocalPravegaEmulator(int zkPort, int controllerPort, int segmentStorePort, int restServerPort,
```

Comment:
Same comment about the constructor as for `InProcPravegaCluster`

Reply:
Using class level lombok builder pattern.